packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md

```diff
@@ -1,60 +1,89 @@
 ﻿# Angular for XMCloud
 
-> Sitecore JSS Angular App for XM Cloud. For the current release this feature is experimental.
+> Sitecore JSS Angular App for XM Cloud. For the current release this feature is in public beta.
+> If you encounter any issues or bugs, please submit a new issue in the [Jss GitHub Repo](https://github.com/Sitecore/jss)
 
-[Documentation](<TODO>)
+[Documentation](https://doc.sitecore.com/xmc/en/developers/jss/latest/jss-xmc/introducing-sitecore-javascript-rendering-sdk.html)
 
-This Single Page Application (SPA) built with Angular is designed to be fully compatible with XM Cloud, incorporating several key add-ons and features to streamline the development process and enable seamless integration. The supported key features are as follows:
+This Single Page Application (SPA) built with Angular is designed to be fully compatible with XM Cloud, incorporating several key add-ons and features to streamline the development process and enable seamless integration.
+
+The Angular XM Cloud integration consists of two parts:
+- `XM Cloud Angular`: simplifies connecting the application to XM Cloud and configuring the integration of multiple composable Sitecore products, provides out of the box helper components.
```

Comment:
"provides out of the box helper components." > "and provides out-of-the-box helper components."